src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,677 @@ description: Learn how to use images in Astro.
+
+In `.astro` files, local images must be imported into the file in order to be used. Remote and `public/` images do not require importing.
+
+Import and use Astro's built-in `<Image />` component for optimized images. Alternatively, Astro syntax supports writing an HTML `<img>` tag directly.
+
+```astro title="src/pages/blog/MyImages.astro"
+---
+import { Image } from 'astro:assets';
+import localBirdImage from ''../../images/subfolder/localBirdImage.png';
+---
+<Image src={localBirdImage} alt="A bird sitting on a nest of eggs."/>
+<Image src="/bird-public-folder.jpg" alt="A bird." width="50" height="50"/>
+<Image src="https://example.com/remote-bird.jpg" alt="A bird." width="50" height="50"/>
+
+<img src={localBirdImage.src} alt="A bird sitting on a nest of eggs.">
+<img src="/images/bird-in-public-folder.jpg" alt="A bird.">
+<img src="https://example.com/remote-bird.jpg" alt="A bird.">
+```
+
+### `<Image />` (`astro:assets`)
+
+Use the built-in `<Image />` Astro component to display optimized versions of your local images and [configured remote images](#authorizing-remote-images).
+
+Images in the `public/` folder, as well as remote images not specifically configured in your project can also be used with the Image component, but will not be processed.
+
+`<Image />` can transform a local or authorized remote image's dimensions, file type, and quality to produce an optimized `<img>` tag to avoid **Cumulative Layout Shift (CLS)**.
+
+:::tip[What is Cumulative Layout Shift?]
+[Cumulative Layout Shift (CLS)](https://web.dev/cls/) is a Core Web Vital metric for measuring how much content shifted on your page during loading. The `<Image />` component optimizes for CLS by automatically selecting the correct `width` and `height` for your local images.
 :::
 
-### In `.astro` files
+```astro title="src/components/MyComponent.astro"
+---
+// import the Image component and the image
+import { Image } from 'astro:assets';
+import myImage from "../assets/my_image.png"; // Image is 1600x900
+---
+
+<!-- `alt` is mandatory on the Image component -->
+<Image src={myImage} alt="A description of my image." />
+```
+
+```html
+<!-- Output -->
+<!-- Image is optimized, proper attributes are enforced -->
+<img
+  src="/_astro/my_image.hash.webp"
+  width="1600"
+  height="900"
+  decoding="async"
+  loading="lazy"
+  alt="A description of my image."
+/>
+```
 
-Astro uses HTML `<img>` elements to display images, and all HTML image attributes are supported.
+Currently, the built-in assets feature does not include a `<Picture />` component.
 
-The `src` attribute is required, and its format will depend on where your images are stored, and whether you have enabled experimental support for assets:
+Instead, you can use the HTML image attributes `srcset` and `sizes` or the `<picture>` tag [for art direction or to create responsive images](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
```

Comment:
You can do that, but since this doesn't teach you about `getImage` yet, from the user's perspective they don't know how to actually use those attributes and tag to use optimized images and multiple sizes at the same time. Using those attributes on `<Image />` won't do anything.